recipes/genv/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: genv
+  version: {{ version }}
+
+source:
+  url: https://github.com/run-ai/genv/archive/v{{ version }}.tar.gz
+  sha256: 9804a967ee4ec1b54febe08d6a67fc5df5b6b04f69c465b4564b9bda983f4e8b
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  run:
+    - python
+
+test:
+  commands:
+    - genv help
+
+about:
+  home: https://github.com/run-ai/genv
+  summary: 'GPU Environment Management'
+  description: |
+    genv lets you easily control, configure and monitor the GPU resources that you are using.
+    It is intendend to improve the life of individual GPU users as well as teams that share multi-GPU machines.
+    This project was highly inspired by pyenv and other version, package and environment management software like Conda, nvm, rbenv.
+  license: AGPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - razrotenberg
```

Comment:
This whole recipe is quite non-standard. What is the reason for that?

Reply:
hi @BastianZim,
first of, thanks for reviewing this PR! 🙏 

I took inspiration from other recipes out there, not really sure which one exactly so I'm having trouble with pointing out some. I might misunderstood some things and implemented this recipe wrongly, but I'll try to explain why I did it this way.

Genv is not a package managed by some package manager like pip, npm, etc.
It is a standalone software and it is installed simply by cloning a Git repository from GitHub.
As Genv is a shell extension (just like Conda, PyEnv, etc.), it should be loaded into the current shell to work, and its `bin/` directory should be added to `PATH` so that `genv` commands would work.

More information about how to install it can be found in the project docs site [here](https://docs.genv.dev/usage/installation.html#terminal).

Because of all this, I found installing it under `/opt` as the most native way of installing such software in Linux.

This is why it is installed under `/opt/genv` in the Conda root directory, and the `genv` executable is symlink-ed from the `bin/` directory of Conda.

Because Genv should be loaded into the current shell, I needed `activate.sh` and `deactivate.sh` so that it would load and unload from the current shell when a Conda environment is activated or deactivated.

Please lmk if there's any more information that I can share here 👌

Reply:
My pleasure!

Ahh ok that makes sense, thanks for the explanation. That is quite an unusual recipe so I'm going to ping some experts here because of the way it is built but that should already be enough info to get us started.

Reply:
@ocefpaf Do you have any experience with these kinds of packages?

Reply:
Not really. But I'm OK merging this one.